src/ngraph/op/dyn_broadcast.hpp

```diff
@@ -0,0 +1,52 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/axis_set.hpp"
+#include "ngraph/op/op.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        /// \brief Operation which "adds" axes to an input tensor, replicating elements from the input as needed along the new axes.
+        /// This is basically the "dynamic shape" version of the static Broadcast op.
+        class DynBroadcast : public Op
+        {
+        public:
+            /// \brief Constructs a conversion operation.
```

Comment:
This comment is incorrect. (Admittedly, I think this exact incorrect comment has been pasted in a few other places. :)) It should describe the actual op.